block/test/BlockSpec.hs

```diff
@@ -0,0 +1,67 @@
+{-# LANGUAGE OverloadedStrings #-}
```

Comment:
Are you aware of the new [code organisation guidelines](https://github.com/input-output-hk/cardano-sl/blob/develop/docs/code-organisation-guidelines.md)? As such the module would be better named `block/test/Test/Pos/Block/Spec.hs`.